client/src/components/ReadMore.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+
+const ReadMore = (props) => (
+  <div>For more information, checkout </div>
```

Comment:
For more information, checkout what? (line 4). Is this an incomplete sentence?

Reply:
This part of the code was edited for troubleshooting purposes. There will be dynamically rendered data refactored here in a future branch.

Reply:
This is fine fort now.